lib/transport/selenium-webdriver/safari.js

```diff
@@ -48,12 +48,12 @@ class SafariDriver extends WebDriver {
     let client;
 
     if (service) {
-      client = service.start().then(url => new HttpClient(url));
+      client = service.start().then(url => new http.HttpClient(url));
```

Comment:
Any reason why chose not to destruct `HttpClient` and `Executor`?

Reply:
`http.HttpClient` is updated after this file is required. So the reference becomes stale hence not using destruct. 

Destruct can be used if we require at [line:49](https://github.com/nightwatchjs/nightwatch/pull/3413/files/a145a8ffa7a49cc1b267141ddf639f9b27dd4c1c#diff-3f5b13c63838cd8124213c65b287f5f733306763e8d8892b625c3e581e3e7469L49). Did not see the need for that.